.gitignore

```diff
@@ -1,2 +1,7 @@
 /node_modules
 /index.js
+/dist
+/.rts2*
+index.esm.js
+index.js
+*.d.ts
```

Comment:
```suggestion
*.d.ts

```

Reply:
Do you definitely want to commit the generated `.d.ts` files? I understand publishing them but usually they don't get committed to the repo.

Reply:
Ah sorry, i forgot to tell, this is just adding a new line at the end of file to conform [this](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file), not deleting that line

Reply:
Ah, sorry for the confusion. Fixed.